.github/README.md

```diff
@@ -57,7 +59,7 @@ progress.</sub></sup><br>
 [here](https://github.com/meshcloud/collie-cli/issues/41) to get updated on
 progress.</sub></sup>
 
-## ☝️ Prerequisites
+## ☝️ Prerequisites<a name="prerequisites"></a>
```

Comment:
What happened here?

Reply:
I added an anchor that I can link from CLI output, otherwise the auto-generated heading anchor contains an url-encoded version of the emoji which is kind of ugly like "#ef8b8b8-prerequisites" and I was afraid that changing the emoji will break the link. hence using an explicit anchor, which i assume is fine since we use html in other places of the markdown already anyway